grammars/html.cson

```diff
@@ -447,7 +447,7 @@
       }
     ]
   'tag-generic-attribute':
-    'match': '(?<=[^=])\\b([a-zA-Z0-9:-]+)'
+    'match': '(?<=[^=]|^)\\b([a-zA-Z0-9:-]+)'
```

Comment:
It seems to work properly for the newline issue.  I made the change and pushed it.  

However, in doing so, I noticed that all versions of this regex will match attribute names that begin with numbers (e.g. `<span 1class="a">`.  From what I can tell it isn't valid html.  Perhaps it should be changed to something like `(?<!=)\\b([a-zA-Z]+[a-zA-Z0-9:-]*)`.  Thoughts?